fabtests/scripts/runfabtests.sh

```diff
@@ -130,12 +130,10 @@ functional_tests=(
 	"fi_recv_cancel -e rdm -V"
 	"fi_unexpected_msg -e msg -I 10"
 	"fi_unexpected_msg -e rdm -I 10"
-	"fi_inj_complete -e msg"
-	"fi_inj_complete -e rdm"
-	"fi_inj_complete -e dgram"
-	"fi_inj_complete -e msg -SR"
-	"fi_inj_complete -e rdm -SR"
-	"fi_inj_complete -e dgram -SR"
+	"fi_msg_flags -e rdm -A inject -v"
+	"fi_msg_flags -e rdm -NR -A inject -v"
+	"fi_msg_flags -e rdm -A inj_complete -v"
+	"fi_msg_flags -e rdm -NR -A inj_complete -v"
```

Comment:
If the test only supports RDM endpoints, we should have the test use that by default and remove needing to pass that in.

Reply:
Yes I agree. RDM is made default in the latest changes.